v2/tools/generator/internal/astbuilder/comments_test.go

```diff
@@ -48,6 +49,9 @@ func TestDocumentationCommentFormatting(t *testing.T) {
 	}
 
 	for _, c := range cases {
+		if c.comment == "" {
+			continue
+		}
```

Comment:
All this does is to suppress the new test case - it's never run at all, so it adds nothing to the coverage.

Reply:
Will update the same